cockroachcloud/cluster-management.md

```diff
@@ -171,16 +171,14 @@ You can use the [**Databases** page](databases-page.html) to create a new databa
 
```

Comment:
Re the page naming, this is up to you. In an ideal world, every page would be titled the same as the file name (or vice versa). However, this might result in a lot of link updates for you. Totally your call! I would understand leaving for now as it's not really in scope for this issue.

Reply:
I'll leave it as is then, since it's not a huge deal!